starknet_devnet/starknet_wrapper.py

```diff
@@ -303,7 +305,23 @@ def __init__(self, starknet_wrapper: StarknetWrapper):
                 self.starknet_wrapper = starknet_wrapper
                 self.preserved_block_info = starknet_wrapper._update_block_number()
 
+            def _check_tx_fee(self, transaction):
+                try:
+                    if (
+                        transaction.version != LEGACY_TX_VERSION
+                        and transaction.max_fee == 0
+                        and not self.starknet_wrapper.config.allow_max_fee_zero
+                    ):
+                        raise StarknetDevnetException(
+                            code=StarknetErrorCode.OUT_OF_RANGE_FEE,
+                            message="max_fee == 0 is not supported.",
+                        )
+                except AttributeError:
```

Comment:
An undefined max_fee is not the only thing that can throw here. I think a clearer way of implementing this would be:

```python
if not hasattr(transaction, "max_fee"):
    return
# continue with checking
```

Reply:
I agree